settings.gradle.kts

```diff
@@ -31,6 +31,12 @@ dependencyResolutionManagement {
     // https://kevinnzou.github.io/compose-webview-multiplatform/installation/
     maven("https://jogamp.org/deployment/maven")
   }
+
+  versionCatalogs {
+    create("kotlinJsWrappers") {
+      from("org.jetbrains.kotlin-wrappers:kotlin-wrappers-catalog:2025.6.4")
+    }
+  }
```

Comment:
This feels really weird vs using our normal version catalog, though this is what's recommended by their docs: https://github.com/JetBrains/kotlin-wrappers#using-in-your-projects